frontend/src/app/features/home/home.component.ts

```diff
@@ -10,6 +12,18 @@ import { SharedModule } from '../../shared/shared.module';
   templateUrl: './home.component.html',
   styleUrl: './home.component.css'
 })
-export class HomeComponent {
+export class HomeComponent implements OnInit{
+  public user: User | null = null;
+  constructor(private readonly authService: AuthService) {}
 
+  ngOnInit() {
+    this.register();
+  }
+
+  private async register(): Promise<User | null> {
+    const user = await this.authService.register();
```

Comment:
Question: how about null ?
Do we need to use await in this function ?

Reply:
Pls recheck